internal/services/cosmos/common/schema.go

```diff
@@ -121,7 +121,7 @@ func CosmosDbIndexingPolicySchema() *pluginsdk.Schema {
 							}
 
 						}
-						return validation.StringInSlice(keys, features.ThreePointOhBeta())
+						return validation.StringInSlice(keys, !features.ThreePointOhBeta())
```

Comment:
We can start removing the `ThreePointOh` and `ThreePointOhBeta` flags
```suggestion
						return validation.StringInSlice(keys, false)
```

Reply:
Thank you. Fixed.